zad5/app/views/artists/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <h2><%= "Tracks by #{@artist.name}" %></h2>
+   <table class="table">
+    <thead>
+      <tr>
+        <th>Title</th>
+        <th class="text-center">Playlists</th>
+        <th class="text-right">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @artist.tracks.each do |track| %>
+        <tr>
+          <td><%= track.title %></td>
+          <td class="text-center">
+            <% track.playlists.each do |playlist| %>
+              <%= link_to playlist.title, playlist %>
+            <% end %>
+          </td>
+          <td class="text-right">
+            <%= link_to 'edit', edit_track_path(track), class: 'btn btn-warning' %>
+            <%= link_to 'remove', track_path(track), method: :delete, class: 'btn btn-danger' %>
```

Comment:
można dodać `data: { confirm: 'Are you sure?' }`, aby przypadkiem nie usunąć niczego 🙈